app/src/api/services/dataExtract.ts

```diff
@@ -0,0 +1,59 @@
+import path from 'path';
+
+import db from '../../db';
+
+interface DataExtractRow {
+    extract_id: number;
+    extracted_on: Date;
+    extract_path: string;
+}
+
+interface DataExtract {
+    extract_id: number;
+    extracted_on: Date;
+    download_path: string;
+}
+
+export async function listDataExtracts(): Promise<DataExtract[]> {
+    try {
+        const extractRecords = await db.many<DataExtractRow>(
+            `SELECT
+                extract_id, extracted_on, extract_path
+            FROM bulk_extracts`
+        );
+
```

Comment:
To consider: apparently `pg-node` converts a `timestamp without timezone` type to a `Date` in the server's local time zone, while we store timestamps in UTC in the database. We probably need to convert the result adequately, a workaround is mentioned here: https://stackoverflow.com/a/38340396

Reply:
From the stackoverflow it looks like the best workaround would be to store `timestamp with timezone` explicitly as UTC.

Reply:
to be honest I am now a bit confused about the handling of time zones in Postgres. From what I've just read in the documentation, `timestamptz` stores UTC values in the database, but upon reading them it assumes that they are in the server's local time zone. What do you then mean, in this context, by storing the dates explicitly as UTC?

Reply:
I guess you mean setting the server time zone explicitly to UTC, upon which `timestamptz` should always be treated as UTC when querying the database.

Reply:
So it seems indeed that the best practice is to keep most columns as `timestamp with time zone` (`timestamptz`). Right now, all our columns in the database are `timestamp without time zone`, and the current time zone set on the production and staging servers is `Europe/London`. I suggest that as part of this PR, we adjust the type of all columns (following [this SO answer](https://dba.stackexchange.com/a/134603) ), change the server time zone to `UTC`, and verify that all the code dealing with timestamps is correct given the new types.

Reply:
Suggestion sounds good :+1: